### ISSUE TITLE
Starting freeipa-server with podman interferes with host SELinux

### ISSUE THREAD

**Issue by spomata** [freeipa/freeipa-container#462]
Hi all,

I have been running freeipa-server:fedora-36 container on a RHEL 8 host machine without any particular issues, however this changed after upgrading the host machine to RHEL 8.6 - which brings in podman v 4.0.2

Now, every time I start the freeipa-server container with a "podman run", the SELinux context for some files in /etc (on the host, outside of the container mounted volume) is changed to container_file_t and I am currently forced to run in permissive mode otherwise I wouldn't even be able to login (one of the affected files is /etc/authselect/nsswitch.conf and it is vital to perform a login).

I already tried to recreate the containers, destroy the server replica and re-enroll it using an updated image freeipa-server image, with no luck. A **restorecon** or a **touch ./autorelabel** will work only until the container is restarted again.
Has anybody any hint on how to further debug this?

Here's the output of "restorecon -RFvvv /etc" for reference:

```shell
[root@ilerda ~]# restorecon -RFvvv /etc

Relabeled /etc/pki/ca-trust/extracted/java/cacerts from unconfined_u:object_r:cert_t:s0 to system_u:object_r:cert_t:s0
Relabeled /etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt from unconfined_u:object_r:cert_t:s0 to system_u:object_r:cert_t:s0
Relabeled /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem from unconfined_u:object_r:cert_t:s0 to system_u:object_r:cert_t:s0
Relabeled /etc/pki/ca-trust/extracted/pem/email-ca-bundle.pem from unconfined_u:object_r:cert_t:s0 to system_u:object_r:cert_t:s0
Relabeled /etc/pki/ca-trust/extracted/pem/objsign-ca-bundle.pem from unconfined_u:object_r:cert_t:s0 to system_u:object_r:cert_t:s0
Relabeled /etc/pki/ca-trust/extracted/edk2/cacerts.bin from unconfined_u:object_r:cert_t:s0 to system_u:object_r:cert_t:s0
Relabeled /etc/authselect/system-auth from system_u:object_r:container_file_t:s0:c206,c359 to system_u:object_r:etc_t:s0
Relabeled /etc/authselect/password-auth from system_u:object_r:container_file_t:s0:c206,c359 to system_u:object_r:etc_t:s0
Relabeled /etc/authselect/fingerprint-auth from system_u:object_r:container_file_t:s0:c206,c359 to system_u:object_r:etc_t:s0
Relabeled /etc/authselect/smartcard-auth from system_u:object_r:container_file_t:s0:c206,c359 to system_u:object_r:etc_t:s0
Relabeled /etc/authselect/postlogin from system_u:object_r:container_file_t:s0:c206,c359 to system_u:object_r:etc_t:s0
Relabeled /etc/authselect/nsswitch.conf from system_u:object_r:container_file_t:s0:c206,c359 to system_u:object_r:etc_t:s0
```

**Comment by adelton** [comment]
I believe you are hitting a RHEL 8.6 podman bug https://bugzilla.redhat.com/show_bug.cgi?id=2083570.

If you are a Red Hat customer, you might want to open a customer case to get the fix backported to RHEL 8.6; otherwise I believe podman SELinux behaviour is planned to get fixed only in RHEL 8.7.

**Comment by spomata** [comment]
Whoops, bad luck then! No, I'm not a Red Hat customer, I guess I will have to wait until podman 4.1 gets in...

Otherwise I could consider moving back to docker but I doubt cgroups v2 support is mature enough to use it with the FreeIPA container, or at least that's what I read in a previous issue here.

Thanks for the prompt response!
